### ISSUE TITLE
Raw associations prevent pagination total being available

### ISSUE THREAD

**Issue by skinofstars** [feathersjs/feathers-sequelize#137]
### Steps to reproduce

Create a before hook association with raw enabled

`hook.params.sequelize = {include: [Model], raw: false}`

Check the resulting pagination, `total: 0`, which should return the actual count.

### Versions

```
"feathers": "^2.1.4",
"feathers-sequelize": "^2.0.1",
"pg": "^6.1.5",
"sequelize": "^3.30.4",
```

**Comment by skinofstars** [comment]
Further investigation shows that Sequelize's `findAndCount`.

https://github.com/sequelize/sequelize/issues/7931

**Comment by DesignByOnyx** [comment]
Yes - I have confirmed that this is a sequelize issue (see the linked issue).  Please continue any conversation on that thread. We should keep this open until the issue is solved.

**Comment by skinofstars** [comment]
Just a little update for anyone else is hitting this problem. A workaround is to add an `after` hook that grabs the count.

```javascript
return hook.service.Model.count().then(count => {
  hook.result.total = count;
  return hook;
});
```

**Comment by daffl** [comment]
Fixed via #162 in v2.3.1
